.github/workflows/main.yml

```diff
@@ -117,7 +117,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        arch: [arm64, x86_64]
```

Comment:
There is a check below for running tests on specific architecture. We still want to verify building crashpad on x86.

Reply:
Updated so that we continue to build for macOS/x64_64. The tests have been disabled as suggested.